outlines/packages.md

```diff
@@ -0,0 +1,32 @@
+# Building a package
+
+1. Why you might want to build a package
+    1. You have a package-based app
+    2. You want to publish some code for people to use in the community
+    3. You want to share code within your organization between different apps
+2. Creating a package using the command line tool
+3. Parts of a Meteor package
+    1. What different parts of package.js mean, point to the docs
+    2. Depending on other packages and build plugins
+         1. You can't depend on a specific version of Meteor; that's not what versionsFrom does
+    3. Adding files and assets
+    4. About architectures
+    5. Semver and the constraint solver
+    6. Cordova plugins
+    7. NPM packages
```

Comment:
I agree that we should tell people how to export sync APIs and maybe link to some code that's out there to help with it. Unless we've given up on this idea.